guides/source/active_job_basics.md

```diff
@@ -131,77 +119,283 @@ That's it!
 
 You can enqueue multiple jobs at once using [`perform_all_later`](https://api.rubyonrails.org/classes/ActiveJob.html#method-c-perform_all_later). For more details see [Bulk Enqueuing](#bulk-enqueuing).
 
-Job Execution
--------------
+Default Backend: Solid Queue
+------------------------------
 
-For enqueuing and executing jobs in production you need to set up a queuing backend,
-that is to say, you need to decide on a 3rd-party queuing library that Rails should use.
-Rails itself only provides an in-process queuing system, which only keeps the jobs in RAM.
-If the process crashes or the machine is reset, then all outstanding jobs are lost with the
-default async backend. This may be fine for smaller apps or non-critical jobs, but most
-production apps will need to pick a persistent backend.
+Solid Queue, which is enabled by default from Rails version 8.0 and onward, is a
+database-backed queuing system for Active Job, allowing you to queue large
+amounts of data without requiring additional dependencies such as Redis.
 
-### Backends
+Besides regular job enqueuing and processing, Solid Queue supports delayed jobs, concurrency controls, numeric priorities per job, priorities by queue order, and more.
```

Comment:
Hi,
Can we use "scheduled jobs" instead "delayed jobs"? as the term "Scheduled jobs" would be less ambiguous than "delayed jobs",
Thank you.

Reply:
This is from the official description of Solid Queue

Reply:
Since the official documentation refers to it as "delayed jobs" I'd prefer to keep it consistent.